hpy/devel/src/runtime/ctx_tracker.c

```diff
@@ -10,10 +10,10 @@
  *    HPyTracker_Add.
  *
  *    Space for this extra handle is created automatically, so
- *    HPyTracker_NewWithSize(ctx, 0) will actually allocated space for one
- *    handle.
+ *    HPyTracker_New(ctx, 2) will actually allocated space for three
```

Comment:
isn't this an implementation detail now that resize is private? From the user point of view, it is enough to know that HPyTracker_Add can fail, and that in that case he/she needs to call HPyTracker_Free, isn't it?

Reply:
It is more of an implementation detail, but the "Calling HPyTracker_New(ctx, n) will ensure that HPyTracker_Add cannot fail for the first n handles" is intended to be a guarantee across implementations.

Maybe that is not a good idea though?